src/components/common/stars.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Instead of creating a custom component, you can use `Rating` component from Material UI: https://material-ui.com/components/rating/#rating